pynailgun/ng.py

```diff
@@ -452,7 +462,7 @@ def _send_command_and_read_response(self, cmd, cmd_args, filearg, env, cwd):
             self._send_tty_format(self.stdin)
             self._send_tty_format(self.stdout)
             self._send_tty_format(self.stderr)
-            for k, v in env.iteritems():
+            for k, v in list(env.items()):
```

Comment:
why is this `list` necessary? I don't see a reason to convert Python 3's generator into a list.

Reply:
Whoops. Fixed and rebased. Thanks!